config/routes.rb

```diff
@@ -0,0 +1,12 @@
+Rails.application.routes.draw do
+  devise_for :users
+  root to: "photos#index"
+  
+  resources :photos do
+    resources :comments, :likes
+  end
+  
+  resources :users 
+    # resources :photos, only: [:index], controller: "user"
+  get  '/users/:user_id/photos', to: 'users#show', as: 'user_photos'
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.